site/docs/foundations/responsiveness/index.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Layout grid
+layout: DetailTechnical
+sidebar:
+  priority: 6
+---
+
+A responsive layout grid is made up of a set of consistently proportioned columns, separated by gutters, and flanked by margins. Basic layout grids are included in the design system to provide a starting point for designing any user interface.
+
+The number of columns and their widths vary depending on the density and breakpoint set. Learn more information about Salt densities in the [density foundation](/salt/foundations/density).
+
+<Image
+  src="/img/foundations/responsiveness-gutters.png"
+  alt="Illustration of columns, gutters and margins."
+/>
+
+| Spacing type   | Definition                                                                                                                       | 
+| -------------- | -------------------------------------------------------------------------------------------------------------------------------- | 
+| Columns        | The space that divides the content of the page into numbers depending on the content width for that breakpoint                   |
+| Gutters        | The equally distributed space between columns that defines the width of UI elements                                              |
+| Margins        | The left and right spacing on the edge of the screen that defines the distance between UI elements and their container bounda    |
+
+## Types of grid
+
+**Fixed grid**: As you shrink from desktop to tablet, the layout remains unchanged, resulting in larger side margins. Text doesn’t wrap, images retain their dimensions, and components have a set width, not adapting to the viewport size. Fixed grid is recommended for web design. 
```

Comment:
what happens when you go the other way, from web to desktop etc ?
Is it greedy and does it always take space or is there a limit ?